src/lighteval/main_vllm.py

```diff
@@ -138,6 +139,8 @@ def vllm(
         generation_parameters = GenerationParameters.from_dict(config)
     else:
         generation_parameters = GenerationParameters.from_model_args(model_args)
+        # We slice out generation_parameters from model_args to avoid double-counting in the VLLMModelConfig
+        model_args = re.sub(r"generation_parameters=\{.*?\},?", "", model_args).strip(",")
```

Comment:
This is a bit hacky, so am open to other ideas!

Reply:
Works for me for now, I trust you debuggued it - but can you create an issue so we don't forget to upgrade to something more robust?

Reply:
Issue here! https://github.com/huggingface/lighteval/issues/579